ocrd-pagetopdf

```diff
@@ -128,34 +133,37 @@ main () {
  
         ocrd log info "found image file '$img_file'"
         if ! output=$(java -jar "$SHAREDIR/ptp/PageToPdf.jar" "${options[@]}" 2>&1); then
-            ocrd log error "PageToPdf failed for ID $in_id (pageId=$pageid): $output"
+            ocrd log error "PageToPdf failed for ID $in_id (pageId=$in_pageId): $output"
             continue
         elif ! [ -f "$out_file" -a -s "$out_file" ]; then
-            ocrd log error "PageToPdf result is empty for ID $in_id (pageId=$pageid): $output"
+            ocrd log error "PageToPdf result is empty for ID $in_id (pageId=$in_pageId): $output"
             continue
         fi
         
         if ((negative2zero)); then
             rm $tmpfile
         fi
 
-        # Add the output file to METS
-        if [ -n "$pageid" ]; then
-            options=( -g $pageid )
-        else
-            options=()
-        fi
-        if [[ "${ocrd__argv[overwrite]}" = true ]]; then
-            options+=( --force )
-        fi
-        options+=( -G $out_file_grp
-                   -m application/pdf
-                   -i "$out_id"
-                   "$out_file" )
         ocrd log info "adding output PDF file '$out_file'"
-        ocrd workspace add "${options[@]}"
-
+        # Add the output file to METS
+        # if [ -n "$in_pageId" ]; then
+        #     options=( -g $in_pageId )
+        # else
+        #     options=()
+        # fi
+        # if [[ "${ocrd__argv[overwrite]}" = true ]]; then
+        #     options+=( --force )
+        # fi
+        # options+=( -G $out_file_grp
+        #            -m application/pdf
+        #            -i "$out_id"
+        #            "$out_file" )
+        # ocrd workspace add "${options[@]}"
```

Comment:
So the output file is no longer added to the METS file?

Reply:
Of course not. See commit msg.

Reply:
Why was that code deactivated instead of simply removed?

Reply:
Because it still illustrates what the more efficient loop-FIFO-bulk-add does.